src/helpers/cleaning_helpers.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+
+def remove_1val_cols(df: pd.DataFrame) -> pd.DataFrame:
+    """Removes columns that contain only a single unique value."""
+    return df.drop(columns=[col for col in df.columns if df[col].nunique() == 1])
+
+def filter_only_lads(df: pd.DataFrame) -> pd.DataFrame:
+    """Filters DataFrame to include only Local Authority District (LAD) data."""
+    df = df[df["geographic_level"] == "Local authority district"]
```

Comment:
is this guaranteed to always be lowercase?

Reply:
Oh, will change